todo-app/src/todo/module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { TodoService } from './service';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Todo } from './entity/todo.entity';
+import { TodoController } from './controller';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([Todo])],
+  controllers: [TodoController],
+  providers: [TodoService],
+})
+export class TodoModule {}
```

Comment:
- todo is the entire application
- so a module that deals with CRUD of individual tasks should be called `tasks` instead
- overall this is a better name